### DISCUSSION TITLE
[SAMPLE] Vote on Proposal X

### DISCUSSION THREAD

**Discussion by briantist** [ansible-community/community-topics#103] [Closed]
Checking out what it might look like to use Discussions Polls for voting instead of issues.

Proposal: 
- https://github.com/ansible-community/community-topics/issues/104

**Comment by samccann** [comment]
Just adding some sample chatter in the comments. Personally, I think tax brackets should come with more bling. Maybe tax refunds can come with a rhinestone set and fabric glue?

**Comment by ssbarnea** [comment]
Considering that I happen to already work for Red Hat, should this count as a reason to get a raise? ;)
